rudder-agent/SOURCES/debug-script/lib/collect

```diff
@@ -105,6 +105,29 @@ collect_files() {
     cp ${RUDDER_OPT}/etc/rudder-networks* "${TARGET_DIRECTORY}/rudder/" 2>/dev/null
     cp ${RUDDER_VAR}/ncf/local/ncf.conf "${TARGET_DIRECTORY}/rudder/" 2>/dev/null
 
+
+    # Hash the passwords
+    # This is ad-hoc, as most files/keys differ
```

Comment:
wouldn't be easier to read if you has a hash_secret() function ?

Reply:
i would pass it the whole regex and the filename ?

Reply:
wouldn't the key name be sufficient ?